jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```diff
@@ -55,15 +55,26 @@ pipeline {
 
         stage('Process Manifest Images') {
             steps {
-                //ToDo Process the manifests CSV to update release image registries for any non GA images (Generates image_mirror_mapping file)
-                echo "Process Manifest Images"
+                dir("${env.WORKSPACE}"){
+                    sh "scripts/process-image-manifests registry-proxy.engineering.redhat.com/rh-osbs/3scale-amp2-3scale-rhel7-operator-metadata:1.11.0-3 3scale 3scale-operator"
+                    sh 'git status'
+                }
             }
         }
 
         stage('Mirror Images') {
             steps {
                 //ToDo Mirror any pre release images to the delorean quay.io account
-                echo "Mirror Images"
+                 dir("${env.WORKSPACE}"){
+                    sh "touch manifests/integreatly-3scale/image_mirror_mapping"
```

Comment:
The lines below can be removed once we've added the creation of the image mirror mappings